cmd/schemagen/schemagen.go

```diff
@@ -0,0 +1,155 @@
+package main
+
+import (
+	"bytes"
+	"embed"
+	"flag"
+	"fmt"
+	"github.com/scylladb/gocqlx/v2"
+	"go/format"
+	"html/template"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"unicode"
+
+	"github.com/gocql/gocql"
+	_ "github.com/scylladb/gocqlx/v2/table"
+)
+
+var (
+	cmd          = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+	flagCluster  = cmd.String("cluster", "127.0.0.1", "a comma-separated list of host:port tuples")
+	flagKeyspace = cmd.String("keyspace", "system", "keyspace to inspect")
+	flagPkgname  = cmd.String("pkgname", "models", "the name you wish to assign to your generated package")
+	flagOutput   = cmd.String("output", "models", "the name of the folder to output to")
+)
+
+var (
+	//go:embed keyspace.tmpl
+	rootFs embed.FS
+)
+
+func main() {
+	assert(cmd.Parse(os.Args[1:]), "can't parse flags")
+
+	schemagen()
+}
+
+func schemagen() {
+	writeToFile(
+		renderTemplate(
+			fetchMetadata(
+				createSession())))
+}
+
+func assert(err error, msg string) {
```

Comment:
Panic here is not great.
I'd encourage you to use `log` package instead of prints and use log.Fatalf here.
This function is not very idiomatic and I'd like to see it inlined with log.Fatalf.